acoustid/db.py

```
@@ -59,7 +59,7 @@ def connection(self, bind_key, read_only=False):
             read_only_bind_key = bind_key + ":ro"
             if read_only_bind_key in self.engines:
                 bind_key = read_only_bind_key
-        return self.session.connection(bind=self.engines[bind_key])
+        return self.session.connection(bind_arguments=dict(bind=self.engines[bind_key]))
```

Ruff (0.8.2)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)